src/components/accordion/Accordion.stories.jsx

```diff
@@ -100,3 +100,32 @@ export const NoGaps = () => (
     </AccordionItem>
   </Accordion>
 );
+
+const CONTROLLED_ACCORDION_IDS = [
+  'accrodion item 1',
+  'accordion item 2',
+  'accordion item 3',
+];
+
+let currentAccordionId = null;
+const setCurrentAccordionId = id => {
+  currentAccordionId = id;
+};
+
+export const Controlled = (args: any) => (
+  <Accordion onChange={setCurrentAccordionId} {...args}>
```

Comment:
hey, just fyi, this shouldn't work :) - calling `setCurrentAccordionId`  has no impact as this is function outside component (it won't cause component to rerender as in useState/useReducer). This code works only because we're overwriting state in accordion reducer.

I will fix this in review pr so no action from your side is required

Reply:
🤦  You are totally right, I overlooked that. I see - it works only because we change `index` prop via controls.